src/react-components/src/__tests__/components/UnderlineLink-test.tsx

```diff
@@ -24,14 +24,14 @@ describe("Link With Underline", () => {
   });
   it("throws an error if text is passed but no url is passed", () => {
     expect(() => Enzyme.mount(<UnderlineLink>Test</UnderlineLink>))
-      .to.throw("IconLink needs prop \"url\"");
+      .to.throw("if children Test has no anchor tag, please pass URL");
   });
   it("Throws error when non-link component passed without URL", () => {
     expect(() => Enzyme.mount(<UnderlineLink><span>Test</span></UnderlineLink>))
-    .to.throw("IconLink needs prop \"url\"");
+    .to.throw("if children [object Object] has no anchor tag, please pass URL");
```

Comment:
`[object Object]` is hard to understand and I think you can get away with just saying "the children", but I would change `URL` to `please pass a 'url' prop`.

Reply:
I'd LOOOOVE to figure out how to print a truncated tostring() because when integrating it we get so many of these kinds of errors and they're very hard to track down.  February.

Reply:
Maybe something like?:
```
const children = React.Children.map(this.props.children, child =>
   (child as JSX.Element).type);

throw new Error(`Please only pass one child, got ${children.join(', ')}`);
```

It'll give you `Error: Please only pass one child, got span, span` for `<UnderlineLink url="#passed-in-link"><span>test</span><span>hello</span></UnderlineLink>`.